### PR TITLE
Github Issues migration

### PR DESCRIPTION
**Pull request by ronso0**
hope I caught all occurences (only grep'ed the source for `launchpad` and `bugs`, didn't verify the output, yet)

### CODE REVIEW

File: theme/templates/pages/support.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h5>{% trans %}Community Forums{% endtrans %}</h5>
     <div>
       <h5>{% trans %}Bug Tracker{% endtrans %}</h5>
       <p>
-      {% with bug_tracker_link="https://bugs.launchpad.net/mixxx/" %}{% trans %}Please report any bugs you find to our <a href=" {{ bug_tracker_link }}">Bug Tracker</a>. The bug tracker gives us a centralized place to pool our information on bugs, and prevents us from getting disorganized and forgetting about them.{% endtrans %}{% endwith %}
+      {% with bug_tracker_link="https://github.com/mixxxdj/mixxx/issues/new" %}{% trans %}Please report any bugs you find to our <a href=" {{ bug_tracker_link }}">Bug Tracker</a>. The bug tracker gives us a centralized place to pool our information on bugs, and prevents us from getting disorganized and forgetting about them.{% endtrans %}{% endwith %}
```
Comment (Swiftb0y) [inline theme/templates/pages/support.html:34]:
```suggestion
      {% with bug_tracker_link="https://github.com/mixxxdj/mixxx/issues" %}{% trans %}Please report any bugs you find to our <a href=" {{ bug_tracker_link }}">Bug Tracker</a>. The bug tracker gives us a centralized place to pool our information on bugs, and prevents us from getting disorganized and forgetting about them.{% endtrans %}{% endwith %}
```

**Comment by daschuer** [comment]
Thank you very much!

**Review by daschuer — APPROVED** [review]

**Comment by daschuer** [comment]
Ups ... it looks like I was to quick with merge. Sorry.

**Comment by ronso0** [comment]
@daschuer can you simply push @Swiftb0y's fix?

**Comment by daschuer** [comment]
Done https://github.com/mixxxdj/website/commit/5a17ff40781380e8fe91656b95e9695cc8957761

**Pull request merged**